src/java.base/share/classes/javax/crypto/Cipher.java

```diff
@@ -483,8 +483,8 @@ private static Transform getTransform(Service s,
      *
      * <p> This method traverses the list of registered security Providers,
      * starting with the most preferred Provider.
-     * A new Cipher object encapsulating the
-     * CipherSpi implementation from the first
+     * A new {@code Cipher} object encapsulating the
+     * {@code CipherSpi} implementation from the first
      * Provider that supports the specified algorithm is returned.
```

Comment:
Since "Provider" is capitalized, I think the intent was that this was the classname, so it should also probably be in an `@code` tag. Alternatively, you could change this to non-capitalized "provider" (w/o the @code tag) and I think it would still be readable (and my vote would be for this).

Reply:
Sure, I will use the non-capitalized "provider" whenever suitable. For some cases, there are "provider" argument, so it looks that {@code provider} is more suitable.